README.md

```diff
@@ -31,8 +31,7 @@ module "postgresql" {
 }
 ```
 
-
-<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

Comment:
the pre-commit hook appears to be broken still

Reply:
Seems fine when I run locally? Forced it to run via `pre-commit run --all-files` and it appears to be utilising the correct hook marks in the readme (`<!-- BEGIN_TF_DOCS -->//<!-- END_TF_DOCS -->`

Reply:
By 'seems fine' what are you seeing?

If I add a new variable it doesn't seem to be adding that to the docs for me

Reply:
Updated configuration.